mantidimaging/core/utility/command_line_arguments.py

```diff
@@ -35,7 +35,7 @@ def _log_and_exit(msg: str):
 
 class CommandLineArguments:
     _instance = None
-    _images_path = ""
+    _images_path = [""]
```

Comment:
Can this be an empty list? It might need a type hint
```
_images_path: list[str] = []
```
Might mean can append straight in without needing the `valid_paths` variable below.

Reply:
I agree. Thanks for the suggestion, have put in a refactor to update this 👍